<!-- Purpose: launch waypoints generation node and turtlesim-->

<launch>
<node name = "roaming_turtle" pkg="turtlesim" type="turtlesim_node" required="true"/>

<node name = "TurtleWay" pkg = "tsim"  type = "TurtleWay" output="screen" >
  <rosparam file="$(find tsim)/config/params.yaml" />
    <remap from="/cmd_vel" to="/turtle1/cmd_vel"/>
</node>

<!--   <arg name="rqt_plot_on" default="true" doc="set this to true if you want to see rqt_plot"/> -->
<!--   <node name="pose_plot" pkg="rqt_plot" type="rqt_plot" args="/pose_error" if="$(arg rqt_plot_on)"/> -->

</launch>
